inst/markdown/compareScenarios2/cs2_main.Rmd

```diff
@@ -105,6 +105,40 @@ dataHist %>%
   filter(period %in% params$yearsHist, !is.na(value)) ->
   dataHist
 
+# filter faulty historic data 
+# (see e.g. https://github.com/pik-piam/mrremind/issues/194)
+# TODO: can be removed once it is being dealt with on the mrremind end
+dataHist <- dataHist %>% 
+  # filter regional cement and steel production data from IEA ETP
+  anti_join(
+    expand_grid(
+      model = grep('^IEA ETP', getModels(dataHist), value = TRUE),
+      region = setdiff(getRegs(dataScen), 'World'),
+      variable = c('Production|Industry|Cement', 
+                   'Production|Industry|Steel')),
+    
+    c('model', 'region', 'variable')
+  ) %>% 
+  # remove regional and global data from IEA WEO 2021
```

Comment:
Can be removed for IEA WEO 2021, since I decided to remove global data from IEA WEO 2021 source until a clean long-term solution is found.

Reply:
MIght as well stay.  Worst case scenario, it's doing nothing.

Reply:
Ok, fine by me. I'll let Christof have the final say.

Reply:
If it is removed eventually and someone is responsible for doing so, I am fine with it.
But it shouldn't stay forever when it does not serve a purpose anymore.